src/sentry/lang/native/symbolicator.py

```diff
@@ -28,18 +30,31 @@ def _task_id_cache_key_for_event(project_id, event_id):
     return f"symbolicator:{event_id}:{project_id}"
 
 
+@dataclass(frozen=True)
+class SymbolicatorTaskKind:
+    is_js: bool = False
+    is_low_priority: bool = False
+    is_reprocessing: bool = False
+
+    def with_low_priority(self, is_low_priority: bool) -> "SymbolicatorTaskKind":
```

Comment:
Why is the return type in quotes? Is it because it's the class being defined?

Reply:
Python can’t refer to the `Self` type directly in the statement that defines it, so using a string is the sanctioned workaround there.